test/fixtures/linter/rules/renderer/ControlRerenderOverride.js

```diff
@@ -0,0 +1,36 @@
+sap.ui.define(["sap/ui/core/Control"], function(Control, Button) {
+	const Example1 = Control.extend("sap.ui.demo.linter.controls.Example1", {
+		metadata: {},
+
+		rerender: function() {
```

Comment:
What about destructuring or assignment?

```
	const Example1 = Control.extend("sap.ui.demo.linter.controls.Example1", {
		metadata: {},

		rerender: function() {
Comment
What about destructuring or assignment?

```
      var rerender = function() {
			console.log("Overriding rerender method");
			return Control.prototype.rerender.apply(this, arguments);
		},
	const Example1 = Control.extend("sap.ui.demo.linter.controls.Example1", {
		metadata: {},

		rerender, // or rerender: rerender
....
```

Reply:
I added support for a simple assignment as shown above. I do not think that we need to handle more complex scenarios.